source/clear-linux/tutorials/wp-install/wp-install.rst

```diff
@@ -3,204 +3,150 @@
-database:
+Click :guilabel:`Let’s go!`. Enter pertinent information in the screen that appears as shown in Figure 3 below.
 
    Database name:       WordPress
 
    Database username:   wordpressuser
 
    Database password:   wp-example
 
-Enter the following values for the database host and the table prefix:
-
    Database host:  localhost
 
    Table prefix:   wp\_
 
-Figure 3 shows the filled out fields.
+Figure 3 shows the completed fields:
 
 .. figure:: figures/wp-install-3.png
     :alt: Database connection details
     :width:     600
 
-    These details are needed for WordPress to connect to the database.
-
-.. note::
+    Figure 3: Information necessary for WordPress to connect to the database.
 
-   When you enter your password into the password field, it will be in clear
-   text and not asterisks.
+Click :guilabel:`Submit` to complete the setup.
 
-After entering all the data for accessing your database, click the
-:guilabel:`Submit` button.
-
-Figure 4 shows the following screen letting you know the communication
-between WordPress and your database has been successfully set up.
+Figure 4 shows the confirmation screen that verifies a successful setup. Click :guilabel:`Run the install`.
 
 .. figure:: figures/wp-install-4.png
     :alt: Successful database connection.
     :width:     600
 
-    This screen shows WordPress was able to connect to the MySQL database.
+    Figure 4: This screen shows WordPress was able to connect to the MySQL database.
 
-Click the :guilabel:`Run the install` button.
 
-Let the installer run until WordPress is fully installed on your system.
+The installer runs until WordPress is fully installed on your system.
 
-Once the installation is completed, you can name your blog and create a
-Wordpress username and password, see figure 5.
+Once the installation is complete, you can name your blog and create a Wordpress username and password. See figure 5:
 
 .. figure:: figures/wp-install-5.png
     :alt: WordPress user creation
     :width:     600
 
-    Provide WordPress the needed information to create a site and a user
-    with the permissions to change it.
-
-.. tip::
+    Figure 5: WordPress site information screen.
 
-   You can copy the password to log in faster.
 
-Enter all the required information and click the
-:guilabel:`Install WordPress` button.
+Enter all the required information and click the :guilabel:`Install WordPress` button.
 
-Once the installation is complete, the initial login screen appears, see
-figure 6:
+Once the installation is complete, the initial login screen appears, as shown in figure 6:
 
 .. figure:: figures/wp-install-6.png
     :alt: WordPress login
     :width:     600
 
-    The WordPress login screen.
+    Figure 6: The WordPress login screen.
 
 Enter your WordPress username and password.
-Check the :guilabel:`Remember me` checkbox, to save your credentials.
+Check :guilabel:`Remember me` to save your credentials.
```

Comment:
Add numbered step: 

#. Check :guilabel:`Remember me` to save your credentials.

Reply:
Done